### PR TITLE
[W5.11][W14-A4]Sun Bangjie

### PR DESCRIPTION
**Pull request by SunBangjie**
Implemented a new command: edit.
Updated User Guide.
Updated existing tests and added new tests for both JUnit tests and I/O tests.
Developer Guide remains unchanged as nothing is needed to be updated.

### CODE REVIEW

File: test/java/seedu/addressbook/commands/EditCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,247 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.*;
```
Comment (mhq199657) [inline test/java/seedu/addressbook/commands/EditCommandTest.java:6]: Hmm It is recommended to import only what it is required.

**Review by mhq199657 — COMMENTED** [review]
Looks good to me!

File: src/seedu/addressbook/commands/EditCommand.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    @Override
+    public CommandResult execute() {
+        try {
+            final ReadOnlyPerson target = getTargetPerson();
+            addressBook.removePerson(target);
+            addressBook.addPerson(toEdit);
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toEdit));
+        } catch (UniquePersonList.PersonNotFoundException pnfe) {
```
Comment (kowshik-sundararajan) [inline src/seedu/addressbook/commands/EditCommand.java:68]: Good job incorporating exception handling.

**Review by kowshik-sundararajan — COMMENTED** [review]
Good job on this activity. Nice work on updating the JUnit tests as well. Some comments added. Please close the PR after reading them.

**Pull request closed without merging**